tap_lever/streams/opportunities.py

```diff
@@ -10,8 +18,122 @@ class OpportunityStream(TimeRangeStream):
     TABLE = "opportunities"
     KEY_PROPERTIES = ["id"]
 
-    CACHE_RESULTS = True
-
     @property
     def path(self):
         return "/opportunities"
+
+    def sync(self, child_streams=None):
+        LOGGER.info('Syncing stream {} with {}'
+                    .format(self.catalog.tap_stream_id,
+                            self.__class__.__name__))
+
+        self.write_schema()
+
+        return self.sync_data(child_streams)
+
+    def sync_paginated(self, url, params=None, child_stream=None):
```

Comment:
`child_streams` should be plural

Reply:
All variations of this are now `child_streams`